buildtools/convert-translations.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import json
+import sys
+from pathlib import Path
+
+import polib
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(
+        description="""
+Script to convert .po and .pot translation files to .json files for i18next.
+Output files are saved to .build/locale/webcomponent/{language}/app.json.
+Existing app.json files are overwritten
+""",
+        epilog="""
+To convert one (or more) specific file:
+    .build/python-venv/bin/python3 buildtools/convert-translations.py ".build/locale/gmf.pot"
+To convert all .po and .pot files in the project:
+    find .build -name "*.po*" | .build/python-venv/bin/python3 buildtools/convert-translations.py
+""",
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
+
+    if sys.stdin.isatty():
```

Comment:
It was to be able to use the pipe 
```
find .build -name "*.po*" | .build/python-venv/bin/python3 buildtools/convert-translations.py
``` 
instead of typing out every file name (while still being able to pass an argument in the standard way) - maybe you know another way to do this?

Reply:
Then we can't do:
 ```
 convert-translations.py `ls -1 *.po`
 ```
 For me, this test is not needed, but finally, I don't know...

Reply:
You are right, it works! I didn't know you could do that. Thanks